pkg/bindings/connection.go

```diff
@@ -144,10 +145,18 @@ func tcpClient(_url *url.URL) Connection {
 	connection := Connection{
 		URI: _url,
 	}
+	dialer := net.Dial
+	if v, found := os.LookupEnv("CONTAINER_PROXY"); found {
+		if proxyURL, err := url.Parse(v); err == nil {
```

Comment:
Seems worth at least warning on an invalid CONTAINER_PROXY environment variable.

Reply:
YES! 
if the  CONTAINER_PROXY is invalid,  error will return.  if dialing to proxy server failed,  warn msg will log.